sdk/python/foundation-models/system/inference/text-generation/llama-files/score/hf-tgi/score.py

```diff
@@ -0,0 +1,460 @@
+            time.sleep(WAIT_TIME)
+
+        logger.info("Server Started")
+        client = Client(LOCAL_HOST_URI, timeout=client_timeout)  # use deployment settings
+        logger.info(f"Created Client: {client}")
+    except Exception as e:
+        raise Exception(f"Error in creating client or server: {e}")
+
+
+"""
+Read about client accepted parameters here:
+    https://github.com/huggingface/text-generation-inference/tree/5a1512c0253e759fb07142029127292d639ab117/clients/python/text_generation
+
+client.__init__():
+    base_url: str,
+    headers: Optional[Dict[str, str]] = None,
+    cookies: Optional[Dict[str, str]] = None,
+    timeout: int = 10,
+
+client:generate():
+    self,
+    prompt: str,
+    do_sample: bool = False,
+    max_new_tokens: int = 20,
+    best_of: Optional[int] = None,
+    repetition_penalty: Optional[float] = None,
+    return_full_text: bool = False,
+    seed: Optional[int] = None,
+    stop_sequences: Optional[List[str]] = None,
+    temperature: Optional[float] = None,
+    top_k: Optional[int] = None,
+    top_p: Optional[float] = None,
+    truncate: Optional[int] = None,
+    typical_p: Optional[float] = None,
+    watermark: bool = False,
+    decoder_input_details: bool = False,
+
+class Parameters(BaseModel):
+    # Activate logits sampling
+    do_sample: bool = False
+    # Maximum number of generated tokens
+    max_new_tokens: int = 20
+    # The parameter for repetition penalty. 1.0 means no penalty.
+    # See [this paper](https://arxiv.org/pdf/1909.05858.pdf) for more details.
+    repetition_penalty: Optional[float] = None
+    # Whether to prepend the prompt to the generated text
+    return_full_text: bool = False
+    # Stop generating tokens if a member of `stop_sequences` is generated
+    stop: List[str] = []
+    # Random sampling seed
+    seed: Optional[int]
+    # The value used to module the logits distribution.
+    temperature: Optional[float]
+    # The number of highest probability vocabulary tokens to keep for top-k-filtering.
+    top_k: Optional[int]
+    # If set to < 1, only the smallest set of most probable tokens with probabilities that add up to `top_p` or
+    # higher are kept for generation.
+    top_p: Optional[float]
+    # truncate inputs tokens to the given size
+    truncate: Optional[int]
+    # Typical Decoding mass
+    # See [Typical Decoding for Natural Language Generation](https://arxiv.org/abs/2202.00666) for more information
+    typical_p: Optional[float]
+    # Generate best_of sequences and return the one if the highest token logprobs
+    best_of: Optional[int]
+    # Watermarking with [A Watermark for Large Language Models](https://arxiv.org/abs/2301.10226)
+    watermark: bool = False
+    # Get generation details
+    details: bool = False
+    # Get decoder input token logprobs and ids
+    decoder_input_details: bool = False
+"""
+
+
+"""
+{
+   "input_data": {
+      "input_string":"the meaning of life is",
+      "parameters":{
```

Comment:
why do we skip top_p parameter for text generation task?
i see that we don't mention top_p in notebook for text generation examples as well!